packages/ckeditor5-utils/tests/manual/duplicated-import/duplicated-import.js

```diff
@@ -7,8 +7,6 @@
 
 import ClassicEditor from '../../../../../docs/_snippets/build-classic.js';
 
-import '@ckeditor/ckeditor5-core/src/editor/editor.js';
```

Comment:
Let's keep this line and remove the import from docs.

Also, I would add a comment that the CKEditor 5 library is loaded via CDN. Hence, we expect to throw the duplication error.